Sources/Testing/Issues/Issue+Recording.swift

```diff
@@ -9,13 +9,24 @@
 //
 
 extension Issue {
-  /// The known issue matcher, as set by `withKnownIssue()`, associated with the
+  /// The known issue context, as set by `withKnownIssue()`, associated with the
   /// current task.
   ///
   /// If there is no call to `withKnownIssue()` executing on the current task,
   /// the value of this property is `nil`.
   @TaskLocal
-  static var currentKnownIssueMatcher: KnownIssueMatcher?
+  static var currentKnownIssueContext: KnownIssueContext?
```

Comment:
This task local could probably now be relocated to the new `KnownIssueContext` type and be renamed `current`.

Reply:
Good idea, done.